specs/core/0_beacon-chain.md

```diff
@@ -606,14 +606,14 @@ We define the following Python custom types for type hinting and readability:
 
 | Name | SSZ equivalent | Description |
 | - | - | - |
-| `SlotNumber` | `uint64` | a slot number |
-| `EpochNumber` | `uint64` | an epoch number |
-| `ShardNumber` | `uint64` | a shard number |
+| `Slot` | `uint64` | a slot number |
+| `Epoch` | `uint64` | an epoch number |
+| `Shard` | `uint64` | a shard number |
```

Comment:
The reason why I named it `ShardNumber` is that I assumed that we will have other ID schema for identifying the shard chain, maybe `ShardID`.

Reply:
I see... I think in terms of the spec, we would treat those as the same type and value. I understand implementation might choose to use two types.